### DISCUSSION TITLE
Re-organize `project: type` labels

### DISCUSSION THREAD

**Discussion by abdonrd** [Qiskit/qiskit.org#2448] [Ideas]
After:
- https://github.com/Qiskit/qiskit.org/issues/2363

---

Now we should re-organize the `project: type` labels.

Right now we have. And I propose the changes below each one.

- [project: challenges](https://github.com/Qiskit/qiskit.org/labels/project%3A%20challenges)
  - We can continue using it, since it is an internal project
- [project: documentation](https://github.com/Qiskit/qiskit.org/labels/project%3A%20documentation)
  - No issues. Should we remove it?
- [project: qiskit.org](https://github.com/Qiskit/qiskit.org/labels/project%3A%20qiskit.org)
  - We can remove it and assume the issues at [Qiskit/qiskit.org](https://github.com/Qiskit/qiskit.org) as their own
- [project: textbook (old version)](https://github.com/Qiskit/qiskit.org/labels/project%3A%20textbook%20%28old%20version%29)
  - No issues. Should we remove it?
  - Assume issues at [qiskit-community/qiskit-textbook](https://github.com/qiskit-community/qiskit-textbook)
- [project: brand book](https://github.com/Qiskit/qiskit.org/labels/project%3A%20brand%20book)
  - No issues. Should we remove it?
- [project: events](https://github.com/Qiskit/qiskit.org/labels/project%3A%20events)
  - Just one issue. Should we remove it?
- [project: textbook beta](https://github.com/Qiskit/qiskit.org/labels/project%3A%20textbook%20beta)
  - We can remove it and use the [Qiskit/platypus](https://github.com/Qiskit/platypus) repository
- [project: web components](https://github.com/Qiskit/qiskit.org/labels/project%3A%20web%20components)
  - We can remove it and use the [Qiskit/web-components](https://github.com/Qiskit/web-components) repository

After this we could end with this:

- Qiskit/qiskit.org
  - [project: challenges](https://github.com/Qiskit/qiskit.org/labels/project%3A%20challenges)
  - With the issues that were in _[project: qiskit.org]_
- Qiskit/platypus
  - With the issues that were in _[project: textbook beta]_
- Qiskit/web-components
- Qiskit/gh-actions

**Comment by techtolentino** [comment]
@abdonrd - this is a nice refactor of our labels, overall 👏🏽 

Notes on the following, in my opinion:
- `project: documentation` - I remember we [discussed](https://github.com/Qiskit/qiskit.org/discussions/2357) having a better way to start organizing our docs, but it seems like it's not gaining any traction. I guess until we arrive at a plan, we can remove it.

- `project: brand book` - I believe this'll be up to @JRussellHuffman 

- `project: textbook (old version)` - this can assume issues at https://github.com/qiskit-community/qiskit-textbook

- `project: events` - I imagine we'll have some events work in the future, but since none of it is prioritized as of now, I'm ok w/ removing


thanks for all this organizing Abdon 💪🏽

**Comment by JRussellHuffman** [comment]
I'm fine removing the `project: brand book`. If/when we update the brand book in the future, we'll just mark it as with the design tag.

**Comment by eddybrando** [comment]
Perfect! I'll include this changes in https://github.com/Qiskit/qiskit.org/issues/2470.

Thanks a lot for this initiative @abdonrd!

**Reply by eddybrando** [reply]
I think we should keep "project: events". There are some issues with this label: https://github.com/Qiskit/qiskit.org/issues?q=+label%3A%22project%3A+events%22+

**Reply by abdonrd** [reply]
I will review the `project: events` because I think that most of them are from Challenges.

**Reply by abdonrd** [reply]
As I imagined... They were all from Challenges. I have already changed the labels!
